oci/pull.bzl

```diff
@@ -418,97 +417,86 @@ alias(
 )
 """
 
+_SINGLE_PLATFORM_IMAGE_ALIAS = """\
+alias(
+    name = "{target_name}",
+    actual = "@{original}",
+    visibility = ["//visibility:public"],
+)
+"""
+
+def _coreutils_label(rctx):
```

Comment:
I don't understand why the delta is so large, the PR description just says you're changing an error to a warning. Is this related?

Reply:
yes, it replaces `sha256sum` with the coreutils equivalent.